andela/caret-bn-frontend
# #170421866 -  view high rated accommodations

#### What does this PR do?
PR  to view high rated accommodations
#### Description of Task to be completed?
- add view high rated components
- add view high rated action
#### How should this be manually tested?
- Clone ft-notifications-pane-168781628 branch
- run `npm i`
- run `npm run dev`
- after login, you will navigate to the home page where you will see high rate accommodations
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[#170421866](https://www.pivotaltracker.com/story/show/170421866)
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-12-30 at 13 41 37" src="https://user-images.githubusercontent.com/47029978/71580637-2d6ce500-2b0a-11ea-8223-09f3d07d888b.png">

#### Questions:
`N/A`
